### PR TITLE
fix: fixed bugs with 404page

### PR DESCRIPTION
**Pull request by NavI-Rap0**
add styles and fix page

### CODE REVIEW

**Comment by AndriiZakharenko** [comment]
👀

**Review by AndriiZakharenko — APPROVED** [review]

File: src/modules/NotFoundPage/NotFoundPage.tsx [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
-  return <h1>Not Found page</h1>;
+export const NotFoundPage: React.FC = () => {
+  return (
+    <div className={styles.notfound}>
+      <NotFoundSVG />
+      <h1 className={styles.notfound__title}>404 - Page Not Found</h1>
+      <p className={styles.notfound__text}>
+        The page you're looking for doesn't exist.
+      </p>
+      <a href="/" className={styles.notfound__link}>
```
Comment (iyehorova) [inline src/modules/NotFoundPage/NotFoundPage.tsx]: Please use <Link to={Path.main} ... instead <a href="/"

**Review by OleksiiKislukhin — APPROVED** [review]
:eyes:

**Review by iyehorova — CHANGES_REQUESTED** [review]
Good job.
Left a comment for a small fix

**Review by iyehorova — APPROVED** [review]
Nice done. Thx

**Pull request merged**